lib/global.dart

```diff
@@ -5,3 +5,7 @@ disabilityMode is to save the state of the toggle from the app drawer.
 it should be removed once a config file is used to save the user preferences
 */
 bool disabilityMode = false;
```

Comment:
Why not put the travelMode in goto_page_new.dart? Otherwise LGTM

Reply:
Like the disabilityMode, the travelMode is a setting/preference that is needed outside the scope of the goto_page_new.dart. A global is easily accessible, but yeah it's an implementation detail that could possibly be more restrictive, if a need to do so arises